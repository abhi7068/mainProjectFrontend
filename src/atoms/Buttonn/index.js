import React from 'react';
import Button from '@material-ui/core/Button';

function Buttonn(props) {
  return (
    <div>
      <Button
        data-testid="Buttonn"
        // eslint-disable-next-line react/prop-types
        onClick={props.handleClick}
        variant="contained"
        size="small"
        color="primary"
      >
        Redeem
      </Button>
    </div>
  );
}

export default Buttonn;
