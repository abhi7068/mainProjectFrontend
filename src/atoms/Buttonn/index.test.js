import React from 'react';
import Buttonn from '../Buttonn';
import { render, cleanup } from '@testing-library/react';
afterEach(cleanup);
it('renders', () => {
  const { getByTestId } = render(<Buttonn handleClick="handleClick"></Buttonn>);
  expect(getByTestId('Buttonn')).toHaveTextContent('Redeem');
});
it('renders', () => {
  const { asFragment } = render(<Buttonn handleClick="handleClick"></Buttonn>);
  expect(asFragment()).toMatchSnapshot();
});
