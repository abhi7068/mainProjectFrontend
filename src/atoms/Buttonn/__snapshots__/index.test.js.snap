// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-containedSizeSmall MuiButton-sizeSmall"
      data-testid="Buttonn"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Redeem
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
